fixtures/generic/120w-chinese-moving-head.json

```diff
@@ -0,0 +1,699 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "120W Chinese moving head",
```

Comment:
This isn't a "generic" fixture -- it's made by the "Hong Yi Stage Lighting Equipment Co", so it should be in the "hong-yi" folder, and have a name like "HY-G100".